it/controllers/pensions/lifetimeAllowances/PensionLumpSumDetailsControllerISpec.scala

```diff
@@ -256,6 +256,82 @@ class PensionLumpSumDetailsControllerISpec extends IntegrationTest with ViewHelp
           formPostLinkCheck(pensionLumpSumDetails(taxYearEOY), formSelector)
           welshToggleCheck(user.isWelsh)
         }
+
+        "render render Your pension lump sum details page with prefilled value for before tax but not tax paid" which {
+          implicit lazy val result: WSResponse = {
+            dropPensionsDB()
+            val pensionsViewModel = aPensionLifetimeAllowanceViewModel.copy(
+              pensionAsLumpSum = Some(LifetimeAllowance(None, Some(newAmount2)))
+            )
+            insertCyaData(pensionsUserDataWithLifetimeAllowance(pensionsViewModel), aUserRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(
+              fullUrl(pensionLumpSumDetails(taxYearEOY)),
+              user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY, validTaxYearList)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+          titleCheck(user.specificExpectedResults.get.expectedTitle, user.isWelsh)
```

Comment:
This common parts of this block should be put in a method and called here and in the test following